src/modules/auth/containers/reset-password/ResetPassword.module.css

```
@@ -16,7 +23,21 @@
   display: flex;
   flex-direction: column;
   row-gap: 16px;
-  align-items: center;
+  align-items: stretch;
+  justify-content: center;
+  gap: 24px;
```

Biome (2.1.2)

[error] Unexpected shorthand property gap after row-gap



(lint/suspicious/noShorthandPropertyOverrides)